### PR TITLE
HIP Bin prefix path updated to rocm_path as per reorg structure

### PR DESCRIPTION
**Pull request by arvindcheru**
Proposed Changes:
- Update HIP bin prefix path to rocm_path (rocm_path/bin will be used for hipcc as per the reorg structure)

### CODE REVIEW

**Comment by kiritigowda** [comment]
@asalmanp can you review this? @arvindcheru is this currently on 5.2.X?

**Comment by arvindcheru** [comment]
> @asalmanp can you review this? @arvindcheru is this currently on 5.2.X?

Changes proposed for 5,3, patch on 5.2.X latest

**Review by asalmanp — APPROVED** [review]

**Review by kiritigowda — APPROVED** [review]

File: amd_openvx_extensions/amd_migraphx/CMakeLists.txt [hunk 1]
```diff
@@ -23,7 +23,7 @@ project(vx_amd_migraphx)
 
 set(CMAKE_CXX_STANDARD 14)
 
-list (APPEND CMAKE_PREFIX_PATH ${ROCM_PATH}/hip ${ROCM_PATH})
+list (APPEND CMAKE_PREFIX_PATH ${ROCM_PATH} ${ROCM_PATH}/hip)
```
Comment (kiritigowda) [inline amd_openvx_extensions/amd_migraphx/CMakeLists.txt:26]: @arvindcheru is this required or can we just have $(ROCM_PATH} as /hip is deprecated?

Comment (arvindcheru) [reply amd_openvx_extensions/amd_migraphx/CMakeLists.txt:26]: Thanks for comments, in the proposed changes we were prioritizing (order) the new path. this is same approach used for search path and not removed old path in current version for all related changes. If there are additional thoughts/comments related to this Please let us know.

**Pull request merged**